character_map_tester.cpp

```diff
@@ -9,24 +9,21 @@
 #include <cctype>
 using std::tolower;
 
-#include <iostream>
-using std::cout;
-using std::endl;
+#include <assert.h>
+ 
 
-int ch_map_problematic_character(int *ch_map) {
+
+void ch_map_tester() {
 
   // sacrificing speed for rigidity.

```

Comment:
No need for that :)

Reply:
The role of the function changed. It used to return the problematic character. In fact, that is why it's name was so different to the file name

Reply:
I mean the comment "sacrificing speed for rigidity". That's pretty much assumed in tests :P